tests/Mf2/ParserTest.php

```diff
@@ -139,6 +139,8 @@ public function testParseEWithWhitespace() {
 	 * @group parseH
 	 */
 	public function testInvalidClassnamesContainingHAreIgnored() {
+		self::expectNotToPerformAssertions();
```

Comment:
So is this a test that succeeds if no assertions are performed? I must be misunderstanding it.

Reply:
I'm more used to an assertion that a string is not present or not equal or not contains, but that works.